src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(n -> Arrays.stream(n.split(SEPARATOR)))
+                .mapToInt(Integer::parseInt)
+                .filter(i -> i % 2 == 0)
+                .min()
+                .orElseThrow(() ->
+                        new RuntimeException("Can't get min value from list: " + numbers));
```

Comment:
exception message doesn't look  like in the task
```suggestion
                        new RuntimeException("Can't get min value from list: < Here is our input " + numbers + " >"));
```

Reply:
![image](https://github.com/mate-academy/jv-stream-github-practice/assets/66474575/0b25e0fc-565a-48a9-a269-695dac62e58b)
I don't agree. Look at common misstakes